client/assets/stylesheets/_wp-components-overrides.scss

```diff
@@ -103,3 +108,10 @@
 		}
 	}
 }
+
+// WordPress.org components no longer use blue-50 as the primary color.
+// This changes the primary color to blue-50 to conform to the WordPress.com colors.
+body {
+	--color-accent: var(--studio-blue-50);
+	--color-accent-60: var(--studio-blue-60);
+}
```

Comment:
This is overriding the pink button styling in calypso, [there's an issue](https://github.com/Automattic/wp-calypso/issues/79772) for essentially doing this but it's a wider scope. 

For now Let's just ship the change to stepper for now (where we don't do logged in user profile themes).

![Screenshot 2023-07-26 at 20-07-53 My Home ‹ Site Title — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/e6b8b6d4-f6cb-4fcd-b044-af570f7f1d68)
![Screenshot 2023-07-26 at 20-07-48 My Home ‹ Site Title — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/4d339a74-1928-4e31-9fd2-f1a0c42ac1c7)